lib/WebService/MinFraud/Validator.pm

```diff
@@ -78,6 +78,11 @@ sub _build_request_schema_definition {
                 optional => {
                     user_agent      => '//str',
                     accept_language => '//str',
+                    session_age     => '//num',
```

Comment:
Any reason not to set the minimum to 0? I think the syntax is something line `range => {min => 0}`.